appAuthHelper.js

```diff
@@ -279,7 +333,7 @@
                     }).bind(this);
 
                     registerServiceWorker().then((function () {
-                        this.identityProxy = {
+                        identityProxy = {
                             tokensRenewed: function (currentResourceServer) {
```

Comment:
It appears that we need to add all of the equivalent support on the service worker side of things